### PR TITLE
Show a gray shield when encrypted by deleted session

### PR DESCRIPTION
**Pull request by SimonBrandner**
Fixes [#13701](https://github.com/vector-im/element-web/issues/13701)

Before:
Screenshot_20210529_133101

After:
Screenshot_20210529_175044

### CODE REVIEW

**Comment by robintown** [comment]
Shouldn't the icon be `e2e/normal.svg` if it's gray?

**Comment by SimonBrandner** [comment]
> Shouldn't the icon be `e2e/normal.svg` if it's gray?

I am not really sure what is the best thing I can do here..

**Comment by robintown** [comment]
Well, it would be in line with how messages appear when their authenticity cannot be verified (and I'm pretty sure that "encrypted by a deleted session" mostly means the same thing)

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -374,7 +374,7 @@ $left-gutter: 64px;
 .mx_EventTile_e2eIcon_unknown {
     &::after {
         mask-image: url('$(res)/img/e2e/warning.svg');
```
Comment (SimonBrandner) [inline res/css/views/rooms/_EventTile.scss:376]:
```suggestion
        mask-image: url('$(res)/img/e2e/normal.svg');
```

**Comment by harmathy** [comment]
I could live with this solution as a fix for [#13701](https://github.com/vector-im/element-web/issues/13701)!

File: res/css/views/rooms/_EventTile.scss [hunk 2]
```diff
@@ -373,8 +373,8 @@ $left-gutter: 64px;
 
 .mx_EventTile_e2eIcon_unknown {
```
Comment (jryans) [inline res/css/views/rooms/_EventTile.scss:374]: While we're here, let's consolidate the blocks, joining the selectors that use the same values. It looks like there's only normal or warning images, so we should only need two blocks.

**Review by jryans — CHANGES_REQUESTED** [review]
Looks fine code wise, but this is really a Product choice here on how much to scream about this case.

**Review by jryans — APPROVED** [review]
Thanks, looks great code wise, so now we need Design input.

**Comment by amshakal** [comment]
Sorry it took me a while get to this. I think this is a reasonable solution.

Overall I think we could a better job of explaining to the user what's going on, what it means for them and what they need to do to resolve a specific encryption related issue in layperson terms but I think that's out of scope for this PR. :D

**Comment by FernOfEdison** [comment]
I think the grey shield is an issue for deleted and unverified sessions.

For sessions that were **unverified and deleted**, the grey shield with "_Encrypted by a deleted session_" still shows. This should not be the case as the source is unverified.

**Comment by richvdh** [comment]
Worth noting we'll be going back to red shields here as part of the change to Element-R (https://github.com/vector-im/element-web/issues/21972)

**Pull request merged**